.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
```

Comment:
Is there a reason why the docker image you've selected is using node 7.10 rather than the latest version?

Reply:
This is what is in the example, but good point, we can update to 12.6.0